Gopkg.toml

```diff
@@ -105,13 +106,17 @@ required = [
 #  name = "k8s.io/ingress-nginx"
 #  revision = "0c6f15e372c831de52fcc393932540bb3a6d51b5"
 
+[[override]]
+  name = "github.com/ugorji/go"
+  revision = "708a42d246822952f38190a8d8c4e6b16a0e600c"
```

Comment:
you seem based on an old toml

please start from the current one in istio/istio master